apps/astarte_trigger_engine/test/amqp_events_consumer_test.exs

```diff
@@ -31,7 +31,10 @@ defmodule Astarte.TriggerEngine.AMQPEventsConsumerTest do
   setup_all do
     :ok = wait_for_connection()
 
-    {:ok, conn} = Connection.open(Config.amqp_consumer_options())
+    {:ok, amqp_consumer_options} = Config.amqp_consumer_options()
```

Comment:
Use bang

Reply:
The bang version of the method doesn't exist. Either we accept the {:ok, value}-tuple as a return value or we implement amqp_consumer_options!()

Reply:
See note on the function definition